keyboards/mariorion_v25/matrix_diagram.md

```diff
@@ -0,0 +1,28 @@
+# Matrix Diagram for MariOrion v2.5
+
```

Comment:
Could you wrap the layout in three backticks to turn it into a codeblock. ([eg](https://github.com/qmk/qmk_firmware/blob/develop/keyboards/alas/matrix_diagram.md?plain=1))